src/puck.ts

```diff
@@ -599,30 +617,51 @@ export class LookupPuck {
 
   unmount(): void {
     removePuck();
-    this.disable();
+    this.setEnabledState(LookupPuckEnabledState.disabled);
     this.puck = undefined;
   }
 
-  enable(): void {
-    this.enabled = true;
-    this.safeAreaProvider.delegate = this;
-    if (this.puck) {
-      this.puck.addEventListener('pointerdown', this.onPuckPointerDown);
+  setEnabledState(enabledState: LookupPuckEnabledState): void {
+    const previousState = this.enabledState;
+    this.enabledState = enabledState;
+
+    if (enabledState === LookupPuckEnabledState.disabled) {
+      this.safeAreaProvider.delegate = null;
+      if (this.puck) {
+        this.stopDraggingPuck();
+        this.puck.removeEventListener('pointerdown', this.onPuckPointerDown);
+      }
+      window.removeEventListener('pointerup', this.noOpPointerUpHandler);
+      this.clickState = { kind: 'idle' };
+      return;
+    }
+
+    // Avoid redoing any of this setup (that's common between both .enableGestures
+    // and .enableGesturesAndLookup).
+    if (previousState === LookupPuckEnabledState.disabled) {
```

Comment:
If preferred, I could just remove the conditional altogether, as the only work we're doing is stuff that is safe to do multiple times (is an effective no-op).